crates/dwallet-classgroups-types/src/lib.rs

```diff
@@ -43,6 +46,8 @@ pub struct DWalletPublicKeys {
     pub public_key: Vec<u8>,
 }
 
+pub const CLASS_GROUPS_SEED_SIZE: usize = 32;
```

Comment:
This isnt the seed size for class group, its the seed size for the random generator algorithm you use (ChaCha20)